langlib/lang.xml/src/main/ballerina/xml.bal

```diff
@@ -131,6 +177,11 @@ public isolated function setName(Element elem, string xName) = @java:Method {
 # This includes namespace attributes.
 # The keys in the map are the expanded names of the attributes.
 #
+# ```ballerina
+# xml:Element a = xml `<person name="John" age="30"></person>`;
+# a.getAttributes() ⇒ {"name":"John","age":"30"}
```

Comment:
Let's use a better example with something that'll actually be an attribute. Name and age are more likely to be items in the XML value.

Reply:
Updated in https://github.com/ballerina-platform/ballerina-lang/pull/38981/commits/8bceaabd7a1a2202d2929fee34d1e15e2f2b9b18